exonum/src/lib.rs

```diff
@@ -23,7 +23,6 @@
     bare_trait_objects
 )]
 #![cfg_attr(feature = "long_benchmarks", feature(test))]
-#![cfg_attr(feature = "cargo-clippy", deny(clippy::clippy_pedantic))]
```

Comment:
Why? Was it that annoying? Or is this switch not supported now?

Reply:
Yes.

Reply:
Which one? ;)

Reply:
Oh, sorry, lint has been renamed to `clippy::pedantic`